.env.example

```diff
@@ -1,4 +1,4 @@
-STAGING=False
+STAGING=
```

Comment:
I don't get this

Reply:
it was an accident. i was in there doing things and i suppose i forgot to add that back.

Reply:
OK, i made a new commit to fix it.